peerconnection_renegotiation_test.go

```diff
@@ -689,15 +687,26 @@ func TestPeerConnection_Renegotiation_NoApplication(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	pcOfferConnected, pcOfferConnectedCancel := context.WithCancel(context.Background())
+	pcOffer.OnICEConnectionStateChange(func(i ICEConnectionState) {
+		if i == ICEConnectionStateConnected {
+			pcOfferConnectedCancel()
+		}
+	})
+
+	pcAnswerConnected, pcAnswerConnectedCancel := context.WithCancel(context.Background())
+	pcAnswer.OnICEConnectionStateChange(func(i ICEConnectionState) {
+		if i == ICEConnectionStateConnected {
+			pcAnswerConnectedCancel()
+		}
+	})
+
 	_, err = pcOffer.AddTransceiverFromKind(RTPCodecTypeVideo, RtpTransceiverInit{Direction: RTPTransceiverDirectionSendrecv})
 	assert.NoError(t, err)
 
 	_, err = pcAnswer.AddTransceiverFromKind(RTPCodecTypeVideo, RtpTransceiverInit{Direction: RTPTransceiverDirectionSendrecv})
 	assert.NoError(t, err)
 
-	// Setting SCTPTransport to nil ensures any interaction with it will cause a segafault
```

Comment:
This is a really stupid way to test this. The test didn't even work because we had nil guards.

Instead I wait until Connected and check the SCTPTransport status